frontend/src/components/OperationDataTable/OperationDataTable.tsx

```diff
@@ -83,15 +60,15 @@ export const OperationDataTable: React.SFC<OperationDataTableProps> = (props) =>
           <Col md={6}>
             Showing {offset + 1} to {list.count() < limit ? offset + list.count() : max}
           </Col>
-          <Col md={6}>
+          <Col md={6} className={classNames({ 'd-none': hidePaginate })}>
             <Pagination className="float-right">
-              <Pagination.First onClick={goToFirst} data-testid="operations-pagination-first">
+              <Pagination.First onClick={first} data-testid="operations-pagination-first">
                 <i className="material-icons">first_page</i>
               </Pagination.First>
-              <Pagination.Prev onClick={goToPrev} data-testid="operations-pagination-prev">
+              <Pagination.Prev onClick={prev} data-testid="operations-pagination-prev">
                 <i className="material-icons">chevron_left</i>
               </Pagination.Prev>
-              <Pagination.Next onClick={goToNext} data-testid="operations-pagination-next">
+              <Pagination.Next onClick={next} data-testid="operations-pagination-next">
                 <i className="material-icons">chevron_right</i>
               </Pagination.Next>
             </Pagination>
```

Comment:
This should be in the wrapper component

Reply:
Also, we created a new and simpler pagination with the `PaginationRow` component.